packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -26,6 +26,15 @@ export class Loan {
   async createLoanScheme (scheme: CreateLoanScheme, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('createloanscheme', [scheme.minColRatio, scheme.interestRate, scheme.id, utxos], 'number')
   }
+
+  /**
+   * List all available loan schemes.
+   *
+   * @return {Promise<LoanSchemeResult[]>}
+   */
+  async listLoanSchemes (): Promise<LoanSchemeResult[]> {
```

Comment:
No pagination?

I guess we might not ever need them.

Reply:
Ya, no pagination